src/Playwright/Contracts/IFrame.cs

```diff
@@ -61,5 +61,14 @@ public partial interface IFrame
 
         /// <inheritdoc cref="SelectOptionAsync(string, IEnumerable{SelectOptionValue}, bool?, float?)" />
         Task<IReadOnlyCollection<string>> SelectOptionAsync(string selector, params IElementHandle[] values);
+
+        /// <inheritdoc cref="WaitForURLAsync(string, Regex, Func{string, bool}, float?, WaitUntilState)"/>
+        Task WaitForURLAsync(string urlString, float? timeout = default, WaitUntilState waitUntil = default);
```

Comment:
Just for context, is there a part of this that should be generated that I'm missing?

Reply:
I would merge this folk. I created #1330 based on a conversation with @pavelfeldman, so we can generate these folks.